backend/app/controllers/spree/admin/orders_controller.rb

```diff
@@ -45,7 +45,7 @@ def index
           params[:q][:completed_at_lt] = params[:q].delete(:created_at_lt)
         end
 
-        @search = if params[:q][:all_stores] == '1'
+        @search = if params[:q][:all_stores] == '1' || !current_store.present?
```

Comment:
@MatthewKennedy let's remove the option to see Orders from all Stores - we don't need that :)